cmd/install/assets/hypershift_operator.go

```diff
@@ -508,18 +508,21 @@ func (o HyperShiftOperatorDeployment) Build() *appsv1.Deployment {
 				Spec: corev1.PodSpec{
 					Affinity: &corev1.Affinity{
 						PodAntiAffinity: &corev1.PodAntiAffinity{
-							RequiredDuringSchedulingIgnoredDuringExecution: []corev1.PodAffinityTerm{
+							PreferredDuringSchedulingIgnoredDuringExecution: []corev1.WeightedPodAffinityTerm{
```

Comment:
The verify test is failing, so it means you need to run locally `make verify` in order to regenerate the `saas_template.yaml` file and add it to the PR.

Reply:
Thank you